test/conformance/ingress/util.go

```diff
@@ -551,6 +538,28 @@ func createService(t *testing.T, clients *test.Clients, svc *corev1.Service) con
 	}
 }
 
+func createExternalNameService(t *testing.T, clients *test.Clients, target, gatewayDomain string) context.CancelFunc {
+	targetName := strings.Split(target, ".")
```

Comment:
splitn? But in general we construct target only to deconstruct it here. May be just pass the 2 parts?

Reply:
done